crypto/rsa/rsa_ameth.c

```diff
@@ -1078,18 +1092,12 @@ static size_t rsa_pkey_dirty_cnt(const EVP_PKEY *pkey)
     return pkey->pkey.rsa->dirty_cnt;
 }
 
-DEFINE_SPECIAL_STACK_OF_CONST(BIGNUM_const, BIGNUM)
-
-static int rsa_pkey_export_to(const EVP_PKEY *from, void *to_keydata,
-                              EVP_KEYMGMT *to_keymgmt, OPENSSL_CTX *libctx,
-                              const char *propq)
+static int rsa_int_export_to(const EVP_PKEY *from, int rsa_type,
```

Comment:
I fail to see where this rsa_type is getting used in this function.. I guess it could do a sanity check maybe?

Reply:
One could argue that we could drop `rsa_type` and trust the call path, but on other hand, we could do with some hardening too...  the latter requires more coding in the rest of the ameth functions, which I think should be done separately, it's not strictly necessary no.

I'm adding a TODO comment on this.